test/interpreter/Variables.test.js

```diff
@@ -100,11 +100,11 @@ describe("interpreter variables", () => {
             },
             {
                 lhs: "double#",
-                values: [str, int32, float, int64],
+                values: [str, int32, int64],
             },
             {
                 lhs: "int64&",
-                values: [str, int32, float, double],
+                values: [str, float, double],
```

Comment:
Can you add a case that proves that we can now safely assign float values to `double#` variables, and int32 values to `int64&` variables?  We're no longer testing that those assignments cause errors, but there's no test to ensure those assignments do work.

Reply:
done